### PR TITLE
Replaced the "\" for a ";"

### PR DESCRIPTION
**Pull request by JoostDkr**
Espeasy Weblog hangs bij placing a  "\" in the log-item.

### CODE REVIEW

File: _P208_Nokia_LCD_5110.ino [hunk 1]
```diff
@@ -5,26 +5,26 @@
...
 //       RST  -      => LCD_pin_1 reset connected to Vcc with 10k resistor
 //       CE  GPIO-5  => LCD_pin_2 chip select
 //       DC  GPIO-32 => LCD_pin_3 Data/Command select
-//       DIN GPIO-23 => LCD_pin_4 Serial data 
-//       CLK GPIO-18 => LCD_pin_5 Serial clock out
-//       In hardware tab; 
-//       - enable SPI; 
-//       - Select VSPI:CLK=GPIO-18, MISO=GPIO-19, MOSI=GPIO-23 
+//       DIN GPIO-23 => LCD_pin_4 Serial data      (Hardware-tab: SPI interface, VSPI-MOSI)
+//       CLK GPIO-18 => LCD_pin_5 Serial clock out (Hardware-tab: SPI interface, VSPI-VLK)
```
Comment (tonhuisman) [inline _P208_Nokia_LCD_5110.ino:24]: VLK is probably CLK?

File: _P208_Nokia_LCD_5110.ino [hunk 2]
```diff
@@ -217,32 +219,37 @@ boolean Plugin_208(byte function, struct EventStruct *event, String& string){
...
+            break;
+          }
         }
       }
   }  // switch (function)
   return success;
 }
 
+void myLog (String message) {
+  addLog(LOG_LEVEL_INFO, "P208: " + message);
```
Comment (tonhuisman) [inline _P208_Nokia_LCD_5110.ino:245]: That's quite inefficient, should be something like:
```
void myLog(const String & message) {
  String log;
  log.reserve(message.length() + 6);
  log += F("P208: ");
  log += message;
  addLogMove(LOG_LEVEL_INFO, log);
}
```

File: _P208_Nokia_LCD_5110.ino [hunk 3]
```diff
@@ -287,10 +294,12 @@ boolean displayText(char deviceTemplate[][48], struct EventStruct *event ){ // 4
             strncpy(html_input[x], newString.c_str(), len);
           }
           lcd3->println(newString);
-          string+=newString+"\\";         
+          logstring+=newString+" ; ";
```
Comment (tonhuisman) [inline _P208_Nokia_LCD_5110.ino:297]: again, inefficient:
```
logstring += newString;
log += F(" ; "); // could probably be: log += ';';
```

File: _P208_Nokia_LCD_5110.ino (hunk 1 above)
Comment (JoostDkr) [reply _P208_Nokia_LCD_5110.ino:24]: Correct!

File: _P208_Nokia_LCD_5110.ino (hunk 2 above)
Comment (JoostDkr) [reply _P208_Nokia_LCD_5110.ino:245]: I'm using the "ESPEasy_ESP32_mega-20211224" the latest version, but I have the compile error "addLogMove was not declared". So I will use your code but still use "addlog"......

File: _P208_Nokia_LCD_5110.ino (hunk 3 above)
Comment (JoostDkr) [reply _P208_Nokia_LCD_5110.ino:297]: I changed the code in your way.

File: _P208_Nokia_LCD_5110.ino (hunk 2 above)
Comment (tonhuisman) [reply _P208_Nokia_LCD_5110.ino:245]: That feature/optimization `addLogMove` was introduced after the 20211224 release, correct, so `addLog` is the best alternative for now.
When developing on ESPEasy, it is advised to use the `mega` brach from the regular ESPEasy repository, that way you also help in beta-testing, and improving, the code, and always have the latest improvements available 😃

**Pull request closed without merging**